src/Instrumentation/SmallInteger.extension.st

```diff
@@ -0,0 +1,26 @@
+Extension { #name : #SmallInteger }
+
+{ #category : #'*Instrumentation' }
+SmallInteger >> insMinus: aNumber [
+	"Primitive. Subtract the argument from the receiver and answer with the
+	result if it is a SmallInteger. Fail if the argument or the result is not a
+	SmallInteger. Essential. No Lookup. See Object documentation
+	whatIsAPrimitive."
+
+	<metaLinkOptions: #( #+ optionDisabledLink )>
+	<noInstrumentation>
+	<primitive: 2>
+	^ super - aNumber
+]
+
+{ #category : #'*Instrumentation' }
+SmallInteger >> insPlus: aNumber [
+	"Primitive. Add the receiver to the argument and answer with the result
+	if it is a SmallInteger. Fail if the argument or the result is not a
+	SmallInteger  Essential  No Lookup. See Object documentation whatIsAPrimitive."
+
+	<metaLinkOptions: #( #+ optionDisabledLink )>
+	<noInstrumentation>
+	<primitive: 1>
+	^ super + aNumber
```

Comment:
This maybe does not need to do super here?

Reply:
Same as insMinus